modules/sts-mode-installing-manual-run-installer.adoc

```diff
@@ -51,7 +51,7 @@ $ openshift-install create manifests
 $ cp /<path_to_ccoctl_output_dir>/manifests/* ./manifests/
 ----
 
-. Copy the private key that the `ccoctl` generated in the `tls` directory to the installation directory:
+. Copy the `tls` directory containing the private key that the `ccoctl` generated to the installation directory:
```

Comment:
. Copy the `tls` directory that contains the private key to the installation directory: